add_license.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
I don't think this should be committed in. This script is useful only once. If some files already have a license header, it will incorrectly add another one

Reply:
removed